app/components/agenda/agenda-overview/agenda-overview-item.hbs

```diff
@@ -104,39 +113,62 @@
         {{/if}}
       {{/if}}
       {{#unless @isEditingFormallyOkStatus}}
+        {{#if @agendaitem.explanation}}
+          <p>
+            <strong class="auk-u-text-bold">
+              {{t "remark-title"}}:
+            </strong>
+            {{@agendaitem.explanation}}
+          </p>
+        {{/if}}
         <div class="auk-hr">
-          <hr/>
+          <hr />
```

Comment:
The explanation is now above the `hr` as seen on the usertest (not in notion) in agenda overview items
Looks a little funny to me (example with documents).
![image](https://user-images.githubusercontent.com/22245223/161952300-013e958a-e78d-4753-a639-b51d06366702.png)
But if this is desired then carry on.

Reply:
Agreed that it looks a little funky, but I took over the styling from the usertest branch. See: https://usertest.kanselarij-dev.s.redpencil.io/vergadering/6225FB46EB3C02DA50165888/agenda/6225FB47EB3C02DA50165889/agendapunten